src/main/java/me/zeroeightsix/kami/module/modules/combat/Auto32k.java

```diff
@@ -307,4 +223,102 @@ public void onUpdate() {
 
     }
 
+    private BlockPos findPlaceAreaInRow(BlockPos placeTestNextPos, int yOffsetSanitized, int range, EnumFacing facingDirection) {
+
+        for (int i = range; i > 0; i--) {
+
+            placeTestNextPos = placeTestNextPos.add(facingDirection.getXOffset(), 0, facingDirection.getZOffset());
+
```

Comment:
formatting in this section could be more compact, i wouldn't bother now but otherwise looks good